test/Microsoft.AspNetCore.Antiforgery.Test/project.json

```diff
@@ -8,6 +8,7 @@
     "Microsoft.AspNetCore.Http": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-*",
```

Comment:
Talked with @davidfowl about when we should apply the netcore platforms dependency and we only need to do it when there's external dependencys included in the project (newtonsoft,autofac, things like that)